### DISCUSSION TITLE
Reproductive overflow [enhancement]

### DISCUSSION THREAD

**Discussion by SelfSealShelf** [MaxRobinsonTheGreat/EvolutionSimulatorV2#59] [Ideas and Suggestions]
Reproduction is currently very "all or nothing", which greatly favors small, space filling organisms. It's a good evolutionary strategy for organisms in the sim to be small, as this blocks the reproduction of larger organisms and simultaneously lowers it's own reproductive cost. 100% of the food an organism collects to reproduce is lost if it fails.

Instead I'd like to suggest a way to make the amount lost configurable. Either a set %, a proportion to size, or an inverse proportion. An inverse proportion in particular would favor smaller organisms, while a proportion to cell count would favor larger ones, especially larger movers.

**Comment by doo158** [comment]
i don't think this would make that huge of a difference. while it does serve as a benefit for bigger creatures, it also benefits the smaller ones, arguably more. what should be added is some sort of massive detriment for smaller creatures, or a massive advantage for larger ones, such to balance it out. maybe i should make a post about it

**Comment by Rocket-Joe** [comment]
You are forgetting that larger creatures also live proportionally longer. This means in environments that are food poor, smaller organisms will die before they find any food at all and large organisms will survive long enough to reproduce.

**Reply by Coolboy-10** [reply]
And they will find the food first

**Comment by aaron-jencks** [comment]
I found the best thing for me that finally worked was to set the brain of the larger organisms to also move towards producers as well as food so that they attack the smaller space efficient organisms
